JDBC/OAuthSampleApp/src/main/java/OAuthSampleApp.java

```diff
@@ -211,41 +204,57 @@ private static void GetAndSetTokens(Map<String, String> formData) throws Excepti
 	    	    String res = buf.toString("UTF-8");
 	        	JsonElement jElement = new JsonParser().parse(res);
```

Comment:
Better to use JsonElement jElement = JsonParser.parseString(res);
JsonParser().parse is deprecated.

Reply:
removed deprecated method use JsonParser().parse(res); used method as suggested.